admin/includes/modules/product_music/collect_info.php

```diff
@@ -80,38 +83,47 @@
   $cInfo = new objectInfo([]);
 }
 
-$artists_array = array(array(
+$artists_array = [
+    [
     'id' => '',
-    'text' => TEXT_NONE));
+    'text' => TEXT_NONE
+    ]
+];
 $artists = $db->Execute("SELECT artists_id, artists_name
                          FROM " . TABLE_RECORD_ARTISTS . "
                          ORDER BY artists_name");
 foreach ($artists as $artist) {
-  $artists_array[] = array(
+  $artists_array[] = [
     'id' => $artist['artists_id'],
-    'text' => $artist['artists_name']);
+    'text' => $artist['artists_name']
+  ];
 }
 
-$record_company_array = array(array(
+$record_company_array = [
+    [
     'id' => '',
-    'text' => TEXT_NONE));
+    'text' => TEXT_NONE
+    ]
+];
 $record_companies = $db->Execute("SELECT record_company_id, record_company_name
                                 FROM " . TABLE_RECORD_COMPANY . "
                                 ORDER BY record_company_name");
 foreach ($record_companies as $record_company) {
-  $record_company_array[] = array(
+  $record_company_array[] = [
     'id' => $record_company['record_company_id'],
-    'text' => $record_company['record_company_name']);
+    'text' => $record_company['record_company_name']
+  ];
 }
 
-$music_genre_array = array(array('id' => '', 'text' => TEXT_NONE));
+$music_genre_array = [['id' => '', 'text' => TEXT_NONE]];
```

Comment:
Please put on separate lines for readability as you did on line 102-107.  Also, please add comma after last entry on lines 105,125,98, 89, etc.

Reply:
ok